lib/user_space/user_x86_64_atomic.h

```diff
@@ -35,15 +35,14 @@
    language instructions (with gcc syntax). "Lock" prefix is used
    everywhere---no optimisation for non-SMP configurations in present.
  */
-
-struct m0_atomic64 {
+typedef struct m0_atomic64 {
 	long a_value;
-};
+} atomic64_t;
```

Comment:
What is the reason to typedef this "atomic64_t"?

It then is used only twice. It is not used in all related functions.
Why is it used in m0_atomic64_add_return() and m0_atomic64_sub_return(), but not in others?